### PR TITLE
GLS-186 Use filtered API endpoints

### PR DESCRIPTION
**Pull request by froddd**
Switches the market guide model to use Directory-API instead of hardcoded data for country trading statistics when new feature switch is enabled: `FEATURE_DATA_PIPELINE_STATS`.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GLS-186
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [x] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/helpers.py [hunk 1]
```diff
@@ -393,21 +394,83 @@ def get_trade_barrier_data(countries_list, sectors_list):
...
+
+
+def get_top_goods_exports_by_country(iso2):
+    response = api_client.dataservices.get_top_five_goods_by_country(iso2=iso2, year=2021)
+
+    return process_top_exports(response)
+
+
+def get_top_services_exports_by_country(iso2):
+    response = api_client.dataservices.get_top_five_services_by_country(iso2=iso2, year=2021)
```
Comment (webbyfox) [inline core/helpers.py:456]: Does this `year` need hardcoded? we probably can default to `2021` in the method parameter?

Comment (froddd) [reply core/helpers.py:456]: Good catch -- it's no longer required so should be removed

**Review by webbyfox — COMMENTED** [review]
Overall looks good, just small comment needs clarification

**Review by webbyfox — APPROVED** [review]
:rocket:

**Pull request merged**